frontend/resources/translations/en.json

```diff
@@ -8,6 +8,10 @@
   "title": "Title",
   "youCanAlso": "You can also",
   "and": "and",
+  "blockActions": {
+    "addBelowTooltip": "Click to add below",
+    "addAboveTooltip": "\nAlt+click to add above"
```

Comment:
Can we build in the new line character into the widget rather than adding it into the i10n string?